vignettes/tutorial.Rmd

```diff
@@ -0,0 +1,722 @@
+---
```

Comment:
I haven't checked what the pdf looks like. Could you please confirm that it paginates ok and looks at least as good as the original?

Reply:
The pdf looks OK on my computer, but does not seem to build when I build locally in R studio. Again, not totally sure what I need to do